pkg/3scale/amp/operator/upgrade.go

```diff
@@ -638,11 +688,11 @@ func (u *UpgradeApiManager) upgradeSidekiqBackendRouteEnv(desired *appsv1.Deploy
 	return reconcile.Result{Requeue: update}, nil
 }
 
-func ensureBackendRouteEnvVar(desired v1.EnvVar, existingEnvVars *[]v1.EnvVar) bool {
+func ensureEnvVar(desired v1.EnvVar, existingEnvVars *[]v1.EnvVar) bool {
```

Comment:
This method belongs to `pkg/helper/envvarutils.go`

Reply:
:+1: I'll add it there

Reply:
Added